Puppetfile

```diff
@@ -30,6 +30,10 @@ mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
 # Plugin modules
 mod 'puppetlabs-azure_inventory', '0.2.0'
+# TODO: Update to forge or puppetlabs namesace
```

Comment:
namesake (?)

Reply:
It is supposed to be `namespace`. I will be removing that comment when I publiish the module to forg.e